### PR TITLE
Nushell.Nushell version 0.34.0

### PR DESCRIPTION
**Pull request by just1a-person**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20800)

### CODE REVIEW

File: manifests/n/Nushell/Nushell/0.34.0/Nushell.Nushell.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+PackageIdentifier: Nushell.Nushell
+PackageVersion: 0.34.0
+PackageLocale: en-US
+Publisher: The Nu Project Contributors
+PackageName: nu
+License: MIT
+ShortDescription: A new type of shell
+Moniker: nushell
+ManifestType: defaultLocale
+ManifestVersion: 1.0.0
```
Comment (KaranKad) [inline manifests/n/Nushell/Nushell/0.34.0/Nushell.Nushell.locale.en-US.yaml:13]:
```suggestion
PackageIdentifier: Nushell.Nushell
PackageVersion: 0.34.0
PackageLocale: en-US
Publisher: The Nu Project Contributors
PackageName: nu
PackageUrl: https://github.com/nushell/nushell
License: MIT
LicenseUrl: https://github.com/nushell/nushell/blob/main/LICENSE
ShortDescription: A new type of shell
Moniker: nushell
Tags:
- shell
- rust
ManifestType: defaultLocale
ManifestVersion: 1.0.0
```
Just adding more metadata

File: manifests/n/Nushell/Nushell/0.34.0/Nushell.Nushell.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+PackageVersion: 0.34.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/nushell/nushell/releases/download/0.34.0/nu_0_34_0_windows.msi
+  InstallerSha256: 76DB00921454EE055F417E19C44C890A7EA0775C84E1FD3C2E0232E0DAA31E38
+  ProductCode: '{B713991D-3589-4BD8-B55D-D8A14EC1A72E}'
+ManifestType: installer
+ManifestVersion: 1.0.0
```
Comment (KaranKad) [inline manifests/n/Nushell/Nushell/0.34.0/Nushell.Nushell.installer.yaml:14]:
```suggestion
PackageIdentifier: Nushell.Nushell
PackageVersion: 0.34.0
MinimumOSVersion: 10.0.0.0
InstallModes:
- interactive
- silent
- silentWithProgress
Commands:
- nu
Installers:
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: msi
  Scope: machine
  InstallerUrl: https://github.com/nushell/nushell/releases/download/0.34.0/nu_0_34_0_windows.msi
  InstallerSha256: 76DB00921454EE055F417E19C44C890A7EA0775C84E1FD3C2E0232E0DAA31E38
  UpgradeBehavior: install
  ProductCode: '{B713991D-3589-4BD8-B55D-D8A14EC1A72E}'
ManifestType: installer
ManifestVersion: 1.0.0
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

File: manifests/n/Nushell/Nushell/0.34.0/Nushell.Nushell.installer.yaml (hunk 2 above)
Comment (itsme-alan) [reply manifests/n/Nushell/Nushell/0.34.0/Nushell.Nushell.installer.yaml:14]: Why is `MinimumOSVersion` required?

Comment (KaranKad) [reply manifests/n/Nushell/Nushell/0.34.0/Nushell.Nushell.installer.yaml:14]: Oh! it was autogenerated using `YamlCreate.ps1` script. I will remove it.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**